server/src/routes/webfinger.rs

```diff
@@ -1,11 +1,26 @@
 use super::*;
-use crate::db::community::Community;
 
 #[derive(Deserialize)]
 pub struct Params {
   resource: String,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct WebFingerResponse {
+  pub subject: String,
+  pub aliases: Vec<String>,
+  pub links: Vec<WebFingerLink>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct WebFingerLink {
+  pub rel: Option<String>,
+  pub r#type: Option<String>,
```

Comment:
Alternatively you can use serde rename which seems cleaner.

https://serde.rs/field-attrs.html

Reply:
Sweet, this worked well. I like it much better.